AutopanoPro.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Categories=Application;Graphics
+Comment=Create panoramics
+Encoding=UTF-8
+Exec=AutopanoPro %u
+TryExec=AutopanoPro
+Icon=AutopanoPro.png
```

Comment:
this is invalid. The icon *name* never include the extension. Also you could as well put the right one.

Reply:
fixed